### DISCUSSION TITLE
Add a Determination Property table

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5832] [Feature Requests]
> Would it be possible to get a Determination property table added to the schema with the next Specify 6 update? It would be very useful for us to have a table just like the preparation property table but for the determinations, so we can track data linked to the determinations that has a 1-to-many framework.
> Currently we are looking at using the Determination Citation table as a workaround, but that has the complication of requiring a link to the Reference Work table.

**Requested By:** Dan at CSIRO ([Asana](https://app.asana.com/0/1201972579963460/1204878371829627/f))

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/8211
